test/helpers/kubectl.go

```diff
@@ -1959,11 +1959,24 @@ func (kub *Kubectl) WaitForKubeDNSEntry(serviceName, serviceNamespace string) er
 // WaitCleanAllTerminatingPods waits until all nodes that are in `Terminating`
 // state are deleted correctly in the platform. In case of excedding the
 // given timeout (in seconds) it returns an error
+
 func (kub *Kubectl) WaitCleanAllTerminatingPods(timeout time.Duration) error {
+	return kub.WaitCleanAllTerminatingPodsInNs("", timeout)
```

Comment:
Nit: Passing "--all-namespaces" here would simplify the code below.
```suggestion
	return kub.WaitCleanAllTerminatingPodsInNs("--all-namespaces", timeout)
```